Scripts/Process/MomentProcess.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace VRTK.Core.Process
+{
+    using UnityEngine;
+    using System;
+
+    /// <summary>
+    /// The MomentProcess is a wrapper for an IProcessable process that has a state to determine when it is to be processed.
+    /// </summary>
+    [Serializable]
+    public sealed class MomentProcess : MonoBehaviour

```

Comment:
Do we need to inherit from `MonoBehaviour`?
- If we don't: Remove it.
- If we do: Remove the `[Serializable]` above.

Reply:
Needs to be `MonoBehaviour` as it has inspector parameters you set in the editor window, so removing `[Serializable]` it is!